sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -567,74 +266,136 @@ export class TopicClient implements Client {
     readonly id: string;
 }
 
-// @public
-export interface TopicDetails {
-    accessedOn?: string;
-    authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
-    createdOn?: string;
-    defaultMessageTtl: string;
-    duplicateDetectionHistoryTimeWindow: string;
-    enableBatchedOperations: boolean;
-    enableExpress?: boolean;
-    enablePartitioning: boolean;
-    enableSubscriptionPartitioning?: boolean;
-    entityAvailabilityStatus?: string;
-    filteringMessagesBeforePublishing?: boolean;
-    isAnonymousAccessible?: boolean;
-    isExpress?: boolean;
-    maxDeliveryCount?: number;
-    maxSizeInMegabytes: number;
-    messageCount?: number;
-    messageCountDetails?: MessageCountDetails;
-    requiresDuplicateDetection: boolean;
-    sizeInBytes?: number;
-    status?: EntityStatus;
-    subscriptionCount?: number;
-    supportOrdering: boolean;
-    topicName: string;
-    updatedOn?: string;
-    userMetadata?: string;
+// @public (undocumented)
+export interface Track2CloseableAsyncIterator<MessageT> extends AsyncIterableIterator<MessageT> {
+    // (undocumented)
+    close(): Promise<void>;
 }
 
-// @public
-export interface TopicOptions {
-    authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
-    defaultMessageTtl?: string;
-    duplicateDetectionHistoryTimeWindow?: string;
-    enableBatchedOperations?: boolean;
-    enablePartitioning?: boolean;
-    maxSizeInMegabytes?: number;
-    requiresDuplicateDetection?: boolean;
-    status?: EntityStatus;
-    supportOrdering?: boolean;
-    userMetadata?: string;
+// @public (undocumented)
+export interface Track2CloseableThing {
+    // (undocumented)
+    close(): Promise<void>;
 }
 
-// @public
-export interface TopicResponse extends TopicDetails {
-    _response: HttpOperationResponse;
+// @public (undocumented)
+export interface Track2FetchOptions {
+    // (undocumented)
+    maxWaitTimeInSeconds?: number;
```

Comment:
Should this have maxMessages?

Reply:
Yes, done (in receiveBatch)

Reply:
It does now (and it's called receiveBatch())